api/src/main/java/io/strimzi/api/kafka/model/balancing/KafkaRebalanceState.java

```diff
@@ -44,9 +44,10 @@ public enum KafkaRebalanceState {
      *     <dt>Rebalancing</dt><dd>While the actual rebalancing is still ongoing</dd>
      *     <dt>Stopped</dt><dd>If the user sets annotation strimzi.io/rebalance=stop.</dd>
      *     <dt>Ready</dt><dd>Once the rebalancing is complete.</dd>
+     *     <dt>ProposalReady</dt><dd>When annotation strimzi.io/rebalance=refresh is applied on rebalance resource and the proposal is updated.</dd>
```

Comment:
question ... if refreshing while Rebalancing, doesn't the FSM pass for ProposalPending? Does it move to ProposalReady directly? I mean ... the FSM depends on the API call to CC, so do you confirm that CC just returns a proposal so it's not pending?

Reply:
It moves to ProposalReady Directly. I have tested this multiple times and it was directly moving to proposal ready

Reply:
Ok so CC API is just returning it's ready.